### DISCUSSION TITLE
Cannot read properties of null (reading '0')

### DISCUSSION THREAD

**Discussion by GronbergInfo** [dani-garcia/vaultwarden#4541] [General]
Hi.
I just the other day installed vaultwarden on my server. 
Exported a bitwarden json (unencrypted) and when i try to import it into my vaultwarden, it throws this error  Cannot read properties of null (reading '0').
Any idea what i need to do to fix this?

Exported BW via vivaldi / web vault, same as im trying to import with.
VW is  Version 2024.1.2

**Comment by BlackDex** [comment]
That message indicates you are not using a https connect. Which would also should have prevented you from logging in at all.

**Reply by GronbergInfo** [reply]
Hi.
VW is behind npm with ssl, doesn't disallow me to create account nor login.

**Reply by BlackDex** [reply]
Then it might be problem with the exported data and something is invalid

**Reply by GronbergInfo** [reply]
When i open the file in NPP. it looks good when i skimmed thru it.
could'nt export thru the vivaldi (chrome) extension, but webvault sent me the json.
